en/releases/1.12.md

```diff
@@ -5,26 +5,65 @@
+
+### Commander
+
+* **Commander: use control mode flags and cleanup arm_disarm ([PR#16266](https://github.com/PX4/PX4-Autopilot/pull/16266))**
+  * Consolidates arming logic, and does a better job at keeping the current state
+* **Commander: Separete out manual control setpoint processing ([PR#16878](https://github.com/PX4/PX4-Autopilot/pull/16878))**
+  * Adds a new class `ManualControl` for handling `manual_control_setpoint` and handles RC loss, RC override, and RC arming/disarming
 
 ### Multicopter
 
-* #### Intuitive stick feel in Position mode
+* **More intuitive stick feel in Position mode**
   * Horizontal stick input mapped to acceleration instead of velocity setpoints
   * Removes unexpected tilt changes upon reaching travel speed velocity
   * Intuitive shunting e.g. when landing
-  * Comparable to Altitude mode but with the familiar benefits
-    * drift less hover
-    * straight fast forward flight
-    * automatic braking when letting go of the stick
   * Opt out possible using [MPC_POS_MODE](../advanced_config/parameter_reference.md#MPC_POS_MODE)
   * Development: [First attempt](https://github.com/PX4/PX4-Autopilot/pull/12072), [Introduction](https://github.com/PX4/PX4-Autopilot/pull/16052), [Improvements](https://github.com/PX4/PX4-Autopilot/pull/16320), [Bugfix zero oscillation](https://github.com/PX4/PX4-Autopilot/pull/16786), [Bugfix position unlock](https://github.com/PX4/PX4-Autopilot/pull/16791), [Bugfix invalid setpoint](https://github.com/PX4/PX4-Autopilot/pull/17078)
 
-* #### Hover thrust independent velocity control gains
-  * Parameters `MPC_{XY/Z}_VEL_{P/I/D}` were replaced with `MPC_{XY/Z}_VEL_{P/I/D}_ACC`, see:<br/>
+* **Hover thrust independent velocity control gains**
+  * Parameters `MPC_{XY/Z}_VEL_{P/I/D}` were replaced with `MPC_{XY/Z}_VEL_{P/I/D}_ACC`, see: 
    [MPC_XY_VEL_P_ACC](../advanced_config/parameter_reference.md#MPC_XY_VEL_P_ACC), [MPC_XY_VEL_I_ACC](../advanced_config/parameter_reference.md#MPC_XY_VEL_I_ACC), [MPC_XY_VEL_D_ACC](../advanced_config/parameter_reference.md#MPC_XY_VEL_D_ACC), [MPC_Z_VEL_P_ACC](../advanced_config/parameter_reference.md#MPC_Z_VEL_P_ACC), [MPC_Z_VEL_I_ACC](../advanced_config/parameter_reference.md#MPC_Z_VEL_I_ACC), [MPC_Z_VEL_D_ACC](../advanced_config/parameter_reference.md#MPC_Z_VEL_D_ACC)
-  * **Attention:** The gains have a new meaning
-    * Scale from velocity error in m/s to acceleration output in m/s^2
-    * Existing gains need to roughly be rescaled by a factor of `* gravitational constant / hover thrust`
-    * Automatic parameter transition assumes 50% hover thrust: ~10m/s^2 / 50% = 20 m/s^2. See [question](https://github.com/PX4/PX4-Autopilot/pull/14823#issuecomment-791357646)
+   
+    :::warning The gains have a new meaning
+    * Scale from velocity error in $m/s$ to acceleration output in $m/s^2$
+    * Existing gains need to roughly be rescaled by a factor of: $gravitational \_ constant / hover\_thrust$
```

Comment:
1. When you say "existing gains" you mean "the old gain" right?
2. FYI only, the addition of `$` around equations above allows them to be rendered using mathjax. This is not compatible with github rendering - i.e. if you ever do this you need to make sure you have tested the rendering works by building locally (which I have done in these cases).

Reply:
1. Yes, this means that they would need to revise their gains after upgrading.